core/arch/arm/kernel/secure_partition.c

```diff
@@ -352,6 +382,30 @@ static TEE_Result sp_dt_get_u32(const void *fdt, int node, const char *property,
 	return TEE_SUCCESS;
 }
 
+static TEE_Result sp_dt_get_u16(const void *fdt, int node, const char *property,
+				uint16_t *value)
+{
+	const fdt32_t *p = NULL;
```

Comment:
I was expecting that it was stored as a 16-bit value in the `fdt`.

Reply:
I have just realized that by default the fdt integers are 32 bits so I should use "boot-order = /bits/ 16 <0>;" for testing. Thanks for the finding, I have corrected it!